app/lib/bindings/events.rb

```diff
@@ -0,0 +1,189 @@
+require 'date'
+require 'active_support/all'
+Pakyow::App.bindings :events do
+	scope :events do
+		restful :events
+
+    options(:parent_group) do
+      get_groups_for_logged_in_person()
+    end
+
+    binding(:parent_group) do
+      bindable.group_id
+    end
+
+		options(:duration) do
+			opts = [[]]
+			opts << [1, "1 hour"]
+			opts << [2, "2 hours"]
+			opts << [3, "3 hours"]
+			opts << [4, "4 hours"]
+			opts << [5, "5 hours"]
+			opts
+		end
+
+    binding(:duration) do
+      bindable.duration
+    end
+
+    options(:venue) do
+      get_venues()
+    end
+
+    binding(:venue) do
+				bindable.venue_id
+    end
+
+    binding(:name) do
+      {
+        :content => bindable.name
+      }
+    end
+
+    binding(:description) do
+      {
+        :content => bindable.description
+      }
+    end
+
+    binding(:start_datetime) do
+      {
+        :content => if bindable.start_datetime then bindable.start_datetime.in_time_zone("Central Time (US & Canada)").strftime('%b %d, %Y %I:%M %p') else "" end
+      }
+		end
+
+		binding(:venue_name) do
+			venue = Venue.where("id = ?", bindable.venue_id).first
```

Comment:
@RandomlyKnighted is correct. Venue_id will always be valid because it is populated from the user selection, which is populated from the list of venues (in the db). So it will always map to a valid `venue_id`